manifests/m/Microsoft/SQLServerManagementStudio/22/Preview/22.0.0/Microsoft.SQLServerManagementStudio.22.Preview.installer.yaml

```diff
@@ -33,12 +33,12 @@ Commands:
 FileExtensions:
 - SQL
 AppsAndFeaturesEntries:
-- DisplayVersion: '22.0.0 Preview 4.0'
+- DisplayVersion: '22.0.0 Preview 5.0'
 Installers:
 - Architecture: x64
-  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/63fecf53-d9f5-41d2-af30-7af6da5608e0/2b79bacf513d46b6693b70f7f072253c817d13044fa2a584307e3926a2a4b814/vs_SSMS.exe
-  InstallerSha256: 2b79bacf513d46b6693b70f7f072253c817d13044fa2a584307e3926a2a4b814
-ReleaseDate: 2025-10-27
+  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/82de2b0e-c5c9-4a16-8adf-658429a1446d/e7301926d7dad80e1875f450882a3c2f44fad389c7f1549dabf89a34e47fd5d5/vs_SSMS.exe
+  InstallerSha256: e7301926d7dad80e1875f450882a3c2f44fad389c7f1549dabf89a34e47fd5d5
+ReleaseDate: 2025-11-03
```

Comment:
does it matter that the release went out on 2025-11-04?

Reply:
Good point. I prepared it on Sunday... Let me fix it.